pyproject.toml

```diff
@@ -37,33 +47,67 @@ optional-dependencies.dev = [
   "pytest",
   "pytest-cov",
   "pytest-timeout",
-  "setuptools",
-  "testfixtures",
-  "tox<5",
-]
-optional-dependencies.test = [
-  "parameterized",
-  "pytest",
-  "pytest-cov",
-  "pytest-timeout",
   "testfixtures",
   "tox<5",
 ]
-optional-dependencies.testing = [
-  "parameterized",
-  "pytest",
-  "pytest-cov",
-  "pytest-timeout",
-  "testfixtures",
-  "tox<5",
-]
-urls.Download = "https://github.com/cpplint/cpplint"
 urls.Homepage = "https://github.com/cpplint/cpplint"
 scripts.cpplint = "cpplint:main"
 
 [tool.setuptools]
 py-modules = [ "cpplint" ]
 include-package-data = false
+dynamic.version = { attr = "cpplint.__VERSION__" }
+
+[tool.pylint.basic]
+include-naming-hint = true
+method-rgx = "[A-Z_][A-Za-z0-9]{2,49}$|__init__|__str__|__contains__"
+function-rgx = "[A-Z_][A-Za-z0-9]{2,49}$|main"
+const-rgx = "[a-zA-Z_][A-Za-z0-9_]{2,49}$"
+variable-rgx = "[a-z_][a-z0-9_]{0,49}$"
+argument-rgx = "[a-z_][a-z0-9_]{0,49}$"
+class-rgx = "[A-Z_][a-zA-Z0-9]+$|basestring|unicode|long|xrange"
```

Comment:
`basestring|unicode|long|xrange` were all Python 2 keywords that stopped being supported on 1/1/2020.  Why do we still need them?!?

Reply:
Good question.